stdlib/internals.ncl

```diff
@@ -25,9 +25,10 @@
       else
           %blame% label,
 
-  "$forall_var" = fun sealing_key polarity label value =>
-      let current-polarity = %polarity% label in
-      if polarity == current-polarity then
+  "$forall_var" = fun sealing_key label value =>
+      let current_polarity = %polarity% label in
+      let {polarity, ..} = %lookup_type_variable% sealing_key label in
```

Comment:
I don't know if it makes a big difference, but destructuring involves a contract application and thus merging. It's a bit less visible, but I would use a more down-to-earth `let var_data = ...` and `var_data.polarity` in such low-level code, which is not very pretty anyway, to be avoid the codepath of merging to be taken.

But honestly it couldn't also not make any noticeable difference at all, I'm not sure.

```suggestion
      let {polarity, ..} = %lookup_type_variable% sealing_key label in
```